V2-PRODUCT-SPEC.md

```diff
@@ -0,0 +1,111 @@
+- Click through for a non profit with a project idea
+- There is a description of the process, what it means when they submit a project, expectations, etc... (copy tbd)
+    - Maybe there is a space to tell people if they want help writing their proposal they can come meet us on a Tuesday
+    evening and we can help them.
+    - It should encourage them to communicate with us via our slack room(s) in Eugene Tech (not email).
+- Click post a new project
+- Arrive at a form 
+- They can submit the following (* = optional)
+    - name of the organization
+    - link to organization website*
+    - stakeholder name
+    - stakeholder email (with a note that it will not be publicly visible)
+    - short project description (length of a tweet?)
+    - link to a longer project description (recommend they make a google doc or something?)
+        - it would be best if we didn't have a giant text field, or try to make a markdown thing
+- An email is sent to the stakeholder email
+    - The email contains a 'magic link' to a page where they can edit their posting
+    - The posting has an expiration of 2 months after it was posted.
+        - If it should be extended we can allow them to just re-post it
+- Once approved (see MVP Studio workflow below), the posting will be publicly visible
+
+### Volunteers
+
+**The primary workflow is to discover projects that seems interesting and reach out to help.**
+
+- Arrive at the homepage
+- Click through to see current projects seeking volunteers
+- Presented with a listing of all currently approved projects that have not started
+    - Each listing shows the following
+        - name of the organization
+        - stakeholder name
+        - short project description
+- Click on a project
+- Arrive at a project details page
+- Shows all information about the project
+- Contains a description of the process they would under go if they want to help a project, 
+expectations, what happens next, etc.... (copy tbd)
+- Contains a form for the volunteer to submit the following information
+    - name
+    - email
+    - role (pick only one)
+        - engineer
+        - product owner
+        - web designer
+        - artist
+    - experience
+        - student
+        - new to the industry
+        - industry professional
+    - Why are you interested (big text field)
+- On form submit 
+    - a thank you page is displayed with details on next steps again
+    - the stakeholder gets an email with the details of the volunteer
+
+### MVP Studio
+
+**The primary workflow for MVP Studio is to change the status of a project**
+
+- A project can be in one of the following statuses
+    - pending
+    - approved
+    - started
+    - finished
+- When a new project is submitted a 'magic link' is sent to team@mvpstudio.org
+    - the password in the magic link is different than the one sent to the stakeholder
+- Link goes to a page showing all the details of the new project.
+- There is a drop down to change the status.

```

Comment:
In addition, there could be a button to remove the project

Reply:
In addition, there needs to be a button to remove the project. Could also have a field to add reasons if we want to notify the NP that their project was rejected.